src/lqcut.jl

```diff
@@ -114,71 +118,86 @@ function lq_drift_time_correction(
     )
```

Comment:
There are still things in the `result` missing in my opinion. What is with the GOF, some debug parameters from the correction, the fit etc? 
I guess it is interesting to look later on at least at some variables and their behavior over time. Right now, nothing apart from the string will be saved.